### DISCUSSION TITLE
v0.7.2

### DISCUSSION THREAD

**Discussion by Achronite** [Achronite/mqtt-energenie-ener314rt#88] [Announcements]
### v0.7.2 4/3/2024

The v0.7.x releases require the following updates that will need to be manually installed if upgrading from v0.6.x or below:
* `gpiod` & `libgpiod`: New dependencies that need to be installed  (e.g raspbian: `sudo apt-get gpiod libgpiod-dev`)
* `node.js`: v18.2.0 or greater

### Added

* [#81](https://github.com/Achronite/mqtt-energenie-ener314rt/issues/81) Auto-retry switch command added for MIHO005, **enabled** by default, disabled by adding `retry: false` into `config.json`.  This functions checks that the resulting monitor message matches the (just) sent command; if it is different it retries the command (indefinitely) 
* [#80](https://github.com/Achronite/mqtt-energenie-ener314rt/issues/80) Update state of application (from github) to MQTT '<topic_stub>/board/1/version' with MQTT Discovery added to show if a software update is available for this application within HA under parent entity `mqtt-energenie-ener314rt` `Version`.  This runs once on startup and then checks at 3:26AM every day.

### Fixed

*  MQTT Discovery: [#85](https://github.com/Achronite/mqtt-energenie-ener314rt/issues/85) The 'main' entity for monitor only devices have been switched back to use entity names (rather than the device name)

<hr /><em>This discussion was created from the release <a href='https://github.com/Achronite/mqtt-energenie-ener314rt/releases/tag/v0.7.2'>v0.7.2</a>.</em>

**Comment by genestealer** [comment]
FYI: [retry auto-retry ](retry: false into config.json) is missing from the example config.json

**Reply by Achronite** [reply]
retry: true is in the example https://github.com/Achronite/mqtt-energenie-ener314rt/blob/master/config_sample.json
